admin/common_auth/models.py

```diff
@@ -49,6 +49,10 @@ class MyUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(auto_now_add=True)
     confirmed = models.BooleanField(default=False)
     osf_id = models.CharField(max_length=5, blank=True)
+    desk_key = models.CharField(max_length=45, blank=True)
```

Comment:
As Matt mentioned [here](https://github.com/CenterForOpenScience/osf.io/pull/5056#discussion_r65406994), the key and key secret are specific to the Desk.com API app, not specific to the user. So no need to store these on the user.